src/e3/anod/checkout.py

```diff
@@ -148,7 +148,7 @@ def update_external(
             if p.status != 0:
                 raise e3.error.E3Error("rsync failed")
         else:
-            if os.path.isdir(os.path.join(url, ".git")):
+            if os.path.exists(os.path.join(url, ".git")):
```

Comment:
Why do you feel this is a better check? I think `isdir` checks that the directory `exists` AND is a directory. So `isdir` seems like a better check to me.

Reply:
Ah apologies. I simply didn't read the original issue 🤦 . Never mind.

Reply:
Actually in that case I suggest adding a comment before this if-statement explaining that `.git` can be a file in the case of submodules because that's a lesser known case.

Reply:
@eliericha addressed in #524 @Nikokrock closed this before I had a change to address your comment. :)